### ISSUE TITLE
Incompatible with CoFHCore (for some reason)

### ISSUE THREAD

**Issue by Desempregago** [RakambdaOrg/OverpoweredMending#74]
### Describe the bug

[MultiMC Log](https://pastebin.com/6WDMVx5k)

[Video Proof](https://streamable.com/0urhno)

### Minecraft version

1.16.5

### Forge version

36.2.23

### Mod version

2.2.3

**Comment by Rakambda** [comment]
Sources of that libraries are not public so I can't really do much.
Best is to mention that on the description page.

**Comment by Hekera** [comment]
The source is visible right here: https://github.com/CoFH/CoFHCore

**Comment by Rakambda** [comment]
Weird that they don't link it on their curse page

**Comment by Hekera** [comment]
Relevant method seems to be here: https://github.com/CoFH/CoFHCore/blob/de1378d4bba14500640dfb4e3d411a39322a6ec5/src/main/java/cofh/core/event/CoreCommonEvents.java#L93

**Comment by Rakambda** [comment]
Yeah seems like they have their own mending improvement. Event [is cancelled](https://github.com/CoFH/CoFHCore/blob/8307b4bc7dda6461b620f3219f21687c286feb45/src/main/java/cofh/core/event/CoreCommonEvents.java#L121) at the end so if Forge decides that OverpoweredMending comes after Cofhcore then nothing can be done about it.

The only solution possible would be to increase the priority on my side, but I don't really know what that would mean for compatibility with other mods. It's like I force to take the lead.

**Comment by Hekera** [comment]
I will discuss this with them.

**Comment by Rakambda** [comment]
The important bit is probably this https://github.com/CoFH/CoFHCore/blob/8307b4bc7dda6461b620f3219f21687c286feb45/src/main/java/cofh/core/event/CoreCommonEvents.java#L116 where it stores xp in a container.

**Comment by Desempregago** [comment]
Well, you could increase the priority just to keep the underlying compatibility until the guys at CoFH do something about it.

**Comment by Rakambda** [comment]
You can try with that version for now. But I'm not a too big fan of setting the priority higher that way.
[OverpoweredMending-1.16.5-2.2.3.zip](https://github.com/RakSrinaNa/OverpoweredMending/files/7963487/OverpoweredMending-1.16.5-2.2.3.zip)
